### DISCUSSION TITLE
Disable visual bell for unbound keyseq

### DISCUSSION THREAD

**Discussion by Jai-JAP** [akinomyoga/ble.sh#360] [Q&A] [answered]
How do i disable visual error bell completely including for unbound key and others if any?

I am seeing the below on pressing unbound key sequence. Is it possible to disable it?
![Error vbell](https://github.com/akinomyoga/ble.sh/assets/78354625/2018a2a3-1af5-4c35-a19f-c383feceb443)

**Comment by akinomyoga** [comment, marked as answer]
```bash
# .blerc

bleopt edit_vbell=
bleopt decode_error_char_vbell=
bleopt decode_error_cseq_vbell=
bleopt decode_error_kseq_vbell=
```

**Reply by Jai-JAP** [reply]
I'll try that out tomorrow. Thanks for the quick response.

**Reply by Jai-JAP** [reply]
That did the trick. Thank you so much.

**Reply by akinomyoga** [reply]
No problem.
